### DISCUSSION TITLE
Attempt Spin Flip for Metropolis-Hastings Method

### DISCUSSION THREAD

**Discussion by John-Hayward** [Part-II-Computational-Physics/.github#54] [Q&A] [answered]
I'm unsure about what exactly we are supposed to implement for the attempt spin flip function. Is it supposed to iterate through the entire lattice and attempt to flip each spin (which I initially assumed) or instead choose a random lattice point and attempt to flip just the one?

**Comment by John-Hayward** [comment, marked as answer]
I think now it's actually the latter as attempting to flip the entire sample is both incredibly slow and doesn't give the correct random behaviour that is being modelled

**Reply by DLHvdH** [reply]
It is indeed the latter. I will make a note that the assignment should be clearer on this.
